From 6b972fdb695ce6f6de255424b197a59bdeb4ff8a Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Sat, 10 Feb 2018 19:33:23 -0500
Subject: [PATCH 01/27] usr bin examples

Might as well put example executables in the user's PATH.
Not doing this leads to File System Standards violations.
Of course, this means gnuradio examples need to be named
uniquely, but it is more helpful for new users.
---
 gr-audio/examples/c++/CMakeLists.txt | 2 +-
 gr-fcd/examples/c++/CMakeLists.txt   | 2 +-
 gr-qtgui/examples/c++/CMakeLists.txt | 2 +-
 gr-uhd/examples/c++/CMakeLists.txt   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

--- a/gr-audio/examples/c++/CMakeLists.txt
+++ b/gr-audio/examples/c++/CMakeLists.txt
@@ -30,6 +30,6 @@
 
 INSTALL(TARGETS
     dial_tone
-    DESTINATION ${GR_PKG_AUDIO_EXAMPLES_DIR}
+    DESTINATION ${GR_RUNTIME_DIR}
     COMPONENT "audio_examples"
 )
--- a/gr-fcd/examples/c++/CMakeLists.txt
+++ b/gr-fcd/examples/c++/CMakeLists.txt
@@ -32,6 +32,6 @@
 
 INSTALL(TARGETS
     fcd_nfm_rx
-    DESTINATION ${GR_PKG_FCD_EXAMPLES_DIR}
+    DESTINATION ${GR_RUNTIME_DIR}
     COMPONENT "fcd_examples"
 )
--- a/gr-qtgui/examples/c++/CMakeLists.txt
+++ b/gr-qtgui/examples/c++/CMakeLists.txt
@@ -50,6 +50,6 @@
 
 INSTALL(TARGETS
   display_qt
-  DESTINATION ${GR_PKG_QTGUI_EXAMPLES_DIR}
+  DESTINATION ${GR_RUNTIME_DIR}
   COMPONENT "qtgui_examples"
 )
--- a/gr-uhd/examples/c++/CMakeLists.txt
+++ b/gr-uhd/examples/c++/CMakeLists.txt
@@ -39,6 +39,6 @@
 
 INSTALL(TARGETS
     tags_demo
-    DESTINATION ${GR_PKG_UHD_EXAMPLES_DIR}
+    DESTINATION ${GR_RUNTIME_DIR}
     COMPONENT "uhd_examples"
 )
